pkgs/applications/radio/ubertooth/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, bluez, libbtbb, libpcap , libusb1
+, udevGroup ? "ubertooth"
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ubertooth";
+  version = "2018-12-R1";
+
+  src = fetchFromGitHub {
+    owner = "greatscottgadgets";
+    repo = pname;
+    rev = version;
+    sha256 = "1da5fsjwc5szclq6h5wznvgwikd1qqv8gbmv38h0fpc12lwn85c4";
+  };
+
+  sourceRoot = "source/host";
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+  buildInputs = [ bluez libbtbb libpcap libusb1 ];
```

Comment:
I suppose to make it build, we also remove bluez on macOS.
Could be done by someone who actually has macOS though.

Reply:
Thanks for this catch. Based on the [Build Guide](https://github.com/greatscottgadgets/ubertooth/wiki/Build-Guide) their are no additional dependencies for macOS. Thus, I would make this optional and perhaps somebody could check this? Otherwise I would mark this as a Linux-only package.

Reply:
It might need some framework like `CoreBluetooth` or so: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/qt-5/modules/qtwebengine.nix#L187

Reply:
The updated PR should now contain `CoreBluetooth` and `IOBluetooth` for Darwin. However, I have no idea if this does the trick.

Reply:
@Mic92: What do you think about this changes? What do you think needs to be done to merge the PR?

Reply:
Because I have no knowledge about macOS and nobody else tested it, I removed the OS-dependent stuff and marked `ubertooth` as Linux package. Of course, PRs to address this are very welcome.